src/char/int_homun.cpp

```diff
@@ -189,10 +189,10 @@ bool mapif_homunculus_load(int homun_id, struct s_homunculus* hd)
 	Sql_GetData(sql_handle, 12, &data, nullptr); hd->int_ = atoi(data);
 	Sql_GetData(sql_handle, 13, &data, nullptr); hd->dex = atoi(data);
 	Sql_GetData(sql_handle, 14, &data, nullptr); hd->luk = atoi(data);
-	Sql_GetData(sql_handle, 15, &data, nullptr); hd->hp = atoi(data);
-	Sql_GetData(sql_handle, 16, &data, nullptr); hd->max_hp = atoi(data);
-	Sql_GetData(sql_handle, 17, &data, nullptr); hd->sp = atoi(data);
-	Sql_GetData(sql_handle, 18, &data, nullptr); hd->max_sp = atoi(data);
+	Sql_GetData(sql_handle, 15, &data, nullptr); hd->hp = static_cast<unsigned int>(strtoul(data, nullptr, 10));
+	Sql_GetData(sql_handle, 16, &data, nullptr); hd->max_hp = static_cast<unsigned int>(strtoul(data, nullptr, 10));
+	Sql_GetData(sql_handle, 17, &data, nullptr); hd->sp = static_cast<unsigned int>(strtoul(data, nullptr, 10));
+	Sql_GetData(sql_handle, 18, &data, nullptr); hd->max_sp = static_cast<unsigned int>(strtoul(data, nullptr, 10));
```

Comment:
```suggestion
	Sql_GetData(sql_handle, 15, &data, nullptr); hd->hp = strtoul(data, nullptr, 10);
	Sql_GetData(sql_handle, 16, &data, nullptr); hd->max_hp = strtoul(data, nullptr, 10);
	Sql_GetData(sql_handle, 17, &data, nullptr); hd->sp = strtoul(data, nullptr, 10);
	Sql_GetData(sql_handle, 18, &data, nullptr); hd->max_sp = strtoul(data, nullptr, 10);
```
Same size. Fine without a cast.